pkg/db/test/db.go

```diff
@@ -137,15 +137,15 @@ func cleanupDB() {
 // While the code is somewhat robust to having existing dbs running, this is not intended to be left in the test code.
 // The goal is that you can run individual tests via your IDE integrations or using the CLI, e.g.
 //   go test -run ^TestRetentionHandler$`
-func EnsureDBReady(ctx context.Context) (error, func()) {
+func EnsureDBReady(ctx context.Context) (func(*testing.T), error) {
```

Comment:
this is technically a breaking change but I have not found any use of this function

https://github.com/search?q=org%3Acontiamo+EnsureDBReady&type=code